semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtils.java

```diff
@@ -0,0 +1,42 @@
+package org.hypertrace.semantic.convention.utils.http;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import java.util.List;
+import org.hypertrace.core.span.constants.RawSpanConstants;
+import org.hypertrace.core.span.constants.v1.Http;
+import org.hypertrace.core.span.constants.v1.OTSpanTag;
+
+/**
+ * Utility class to fetch http span attributes
+ */
+public class HttpSemanticConventionUtils {
+
+  // otel specific attributes
+  private static final String OTEL_HTTP_METHOD = OTelHttpSemanticConventions.HTTP_METHOD.getValue();
+  private static final String OTEL_HTTP_STATUS_CODE = OTelHttpSemanticConventions.HTTP_STATUS_CODE.getValue();
```

Comment:
status code related changes are not used, right?

Reply:
Yes, they will be used in next pr.